_posts/en/2020-12-04-better-tech-through-micro-procurementpt2.md

```diff
@@ -0,0 +1,53 @@
+
+There is another challenge related to procurement policy.
+We know that flexibility exists within GC procurement policy to establish a simplified process for LDV procurements.
+However, this flexibility exists because policy doesn't explicitly say it cannot be done.
+As a result, there is very little guidance on how to do micro-procurement within the policy documents or at the departmental level.
+The Office of the Procurement Ombudsman noted this in their [2019 Low Dollar Value Report](http://opo-boa.gc.ca/faiblevaleur-dollarvalue-eng.html) stating that "The audit found that despite adopting the Government of Canada’s contracting policies, TBS's internal process was not supported by a sufficiently articulated policy framework specific to LDV contracting "particularly for non-expert contracting authorities."
+
+Departmental procurement rules can also create barriers to establishing micro-procurement initiatives in the GC.
+Unlike policy, departmental procurement rules can vary widely from department to department.
+This is especially true of rules around credit card use.
+Policy doesn't dictate things like credit card limits and individual transaction limits, but departmental rules do.
+When looking into establishing a micro-procurement initiative in the GC, a good first step would be to learn your department's rules around credit card use.
+
+On the IT side, in order for micro-procurement of digital work to be possible, we must break our work out into small chunks.
+This creates a challenge because we aren't used to working like that.
+We have been grouping IT tasks together into large IT projects for many years.
+ESDC's draft [Target Solution Delivery Model](https://sara-sabr.github.io/ITStrategy/strategy-target-solution-delivery-model.html) suggests that "In order to better deliver services to Canadians, we need to improve IT’s responsiveness, otherwise the feedback loop between policy makers and Canadians will be too long, affecting ESDC’s ability to iterate on its policy making and become an agile organization."
+As we move towards an agile/DevOps way of working, with same day deployments, the need for smaller work packages (and being able to procure these smaller work packages) increases.
+
+Lastly, and certainly not least, any systems change requires breaking down silos and working together towards that change.
+Setting up a process and system for micro-procurement of digital work, by necessity, involves folks from both the CIO and the CFO branches of a department.
+Working together across branches in the GC can be hard.
+We need to learn to speak each other’s languages, and we need to be flexible.
+
+## Our plans at ESDC
+
+Understanding these challenges has helped us move forward with a [Micro-Acquisition Pilot](https://sara-sabr.github.io/ITStrategy/micro-acquisition-pilot.html) at ESDC.
+We are extremely lucky at ESDC to have an amazing procurement innovation team who we (IT Strategy) are working with in partnership on the pilot.
```

Comment:
"at ESDC", repetition from previous sentence? But if you want to keep the emphasis, that's all good :)

Reply:
Yes good point! Fixed.